src/backend/tests/change-requests.test.ts

```diff
@@ -12,6 +11,8 @@ import {
 } from './test-data/change-requests.test-data';
 import { project1 } from './test-data/projects.test-data';
 import { CR_Type } from '@prisma/client';
+import ChangeRequestService from '../src/services/change-request.services';
+import { AccessDeniedException, HttpException, NotFoundException } from '../src/utils/errors.utils';
 
 const app = express();
```

Comment:
we shouldnt need the app anymore as we should now just be testing the service and not the controller